flutter_common/test/data/serialization_test.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:mlperfbench_common/data/results/backend_info.dart';
+
+void main() {
+  group('json serialization', () {
+    final example = BackendReportedInfo(
+      acceleratorName: '1',
+      backendName: '2',
+      filename: '3',
+      vendorName: '4',
+    );
```

Comment:
I prefer to test the whole generated `results.json` file. So if anyone refactors the code, like rename a field, the test would fail and we know that a migration is needed.